apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -209,11 +209,11 @@
       },
       "speedUp": {
         "title": "Speed Up",
-        "description": "Re-submit the transaction with a higher network fee."
+        "description": "Re-submit the transaction with a higher network fee.\n\nThe network fee must be at least 30% higher than the fee of the original transaction."
```

Comment:
Is it ok to hardcode 30%? This isn't something that we can get dynamically?

Reply:
> Is it ok to hardcode 30%?

Not really.

> This isn't something that we can get dynamically?

As with most things, it's probably possible. But after some though, it might not be that interesting to do so at this stage. The upside of displaying this % increase is quite low (especially since a default sensible fee value is automatically applied and if the user wants to set the fees manually, invalid options are disabled) compared to the added cost and complexity of dynamically displaying this value, which is itself based on 2 different env variables (depending on the tx type) in both LLD and LLM.

What I would recommend at this stage is to keep it simple and stay with a generic description, such as:

```suggestion
        "description": "Re-submit the transaction with a higher network fee."
```

And iterate on this once this feature is in customers hands.
This simple message will probably be enough and we can always add a dynamic and more accurate % value latter on if really needed.

Reply:
Just my 2 cent here for next iterations but I feel like these kind of errors should guide the user on what actions needs to be made. So they would see something like `the fees must be at least xxx ETH` instead of `the fees must be at leat x% of the current network fees`. The latter asks more effort as we need to look for the current network fees somewhere and do a calculation.